src/logger.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import winston from 'winston';
 
-export default winston.createLogger({
+export default winston.createLogger<any>({
```

Comment:
What does this do?

Reply:
keep flow from complaining, tried fixing it with the actual type from winston, did not work